core/lib/protobuf_config/src/api.rs

```diff
@@ -151,6 +152,8 @@ impl ProtoRepr for proto::Web3JsonRpc {
                 .context("whitelisted_tokens_for_aa")?,
             extended_api_tracing: self.extended_api_tracing.unwrap_or_default(),
             api_namespaces,
+            deployment_allowlist: read_required_repr(&self.deployment_allowlist)
```

Comment:
This is incorrect at least semantically; `read_required_repr` implies that whatever it reads *is* required. (It may also be incorrect in practice if field conversions inside `DeploymentAllowlist` are fallible; in this case, a conversion error would be incorrectly swallowed.) Please use `read_optional_repr` instead.

Reply:
oh I see. I haven't realized this such function exists.